aten/src/ATen/native/mkldnn/Utils.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <c10/util/ArrayRef.h>
+#include <vector>
+
+namespace at { namespace native {
+
+std::vector<int64_t> expand_param_if_needed(
```

Comment:
are these the same as the other Conv implementation versions?  Can we just combine them?